froide/helper/auth.py

```diff
@@ -224,6 +225,15 @@ def get_user_filter(request, teams=None, fk_path=None):
     return filter_arg
 
 
+def require_staff(view_func):
+    def decorator(request, *args, **kwargs):
+        if not hasattr(request, "user") or not request.user.is_staff:
+            raise PermissionDenied
```

Comment:
Shouldn't `request.user` always be set? https://docs.djangoproject.com/en/4.2/ref/request-response/#attributes-set-by-middleware